components/vf-smooth-scroll/README.md

```diff
@@ -0,0 +1,47 @@
+# Smooth scroll utility component
+
+[![npm version](https://badge.fury.io/js/%40visual-framework%2Fvf-smooth-scroll.svg)](https://badge.fury.io/js/%40visual-framework%2Fvf-smooth-scroll)
+
+## About
+
+This enables smooth scrolling when a user clicks on anchor links or other in-page scrolling is triggered.
+
+## Usage
+
+This is an opt-in behaviour that is activated by putting the `vf-smooth-scroll` class on a parent element — a good place would be at `<html class="vf-smooth-scroll">` (note: it probably won't work if you add it your `<body>` element).
+
+Not that as of November 2021, CSS-based smooth scrolling [is still not support by Safari on Mac OS or iOS](https://caniuse.com/css-scroll-behavior).
```

Comment:
Seems to be working on safari desktop https://stackoverflow.com/questions/19012495/smooth-scroll-to-div-id-jquery/61918221?noredirect=1#comment123675898_61918221

Reply:
I know Safari desktop's technical preview supports it, perhaps it's that?

Reply:
not sure, My assumption is based on SO comment